### PR TITLE
Link to PHP UA blocking from Bots and Indexing

### PR DESCRIPTION
**Pull request by EdwardAngert**
## Summary

**[Bots and Indexing](https://pantheon.io/docs/bots-and-indexing#indexing-your-pantheon-site)** - Adds note about disrespectful bots and crawlers that don't abide by robots.txt, points to [Optimize Site Traffic](https://pantheon.io/docs/optimize-site-traffic#block-user-agents-in-drupal-or-wordpress) for CMS-level option. [PR 5739](https://github.com/pantheon-systems/documentation/pull/5739)

## Effect
(Use this section to detail the changes summarized above, or remove if not needed)
The following changes are already committed:

- link bots and indexing to optimize site traffic doc, remind users that not all bots care about house rules
- add petalbot to robots example

## Remaining Work

The following changes still need to be completed:

- [x] confirm PHP "or"

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

File: source/content/optimize-site-traffic.md [hunk 1]
```diff
@@ -167,12 +167,12 @@ Install and use one of the following WordPress plugins:
...
 
-The `stripos` function implements a case-insenstive match which can be helpful when dealing with mixed bots or crawlers, such as `Curl/dev` vs `curlBot`. 
+The `stripos` function implements a case-insenstive match which can be helpful when dealing with mixed bots or crawlers, such as `Curl/dev` vs `curlBot`.
 
-Remember to replace the example user agent (`UglyBot`):
+Remember to replace the example user agents (`UglyBot` and `PetalBot`):
 
 ```php:title=wp-config.php%20or%20settings.php
-if (stripos($_SERVER['HTTP_USER_AGENT'], 'UglyBot') !== FALSE) {
+if (stripos($_SERVER['HTTP_USER_AGENT'], 'UglyBot' || 'PetalBot') !== FALSE) {
```
Comment (mbaynton) [inline source/content/optimize-site-traffic.md:175]: We should probably just make a loop here if we want an example that shows blocking multiple user agents by substring. There's one-liner ways to do it but they're fairly obtuse and I'd vote for verbosity and clarity in an example like this.

Was this form tested?

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-5739](https://pr-5739--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `52` | `52` | `0` |
| Best Practice Score | `79` | `79` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `98` | `98` | `0` |

Lighthouse Audit Reports:

- [`pr-5739` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/21983/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/21983/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-5739--pantheon-docs.my.pantheonfrontend.website/docs)

File: source/content/optimize-site-traffic.md (hunk 1 above)
Comment (EdwardAngert) [reply source/content/optimize-site-traffic.md]: I updated it with an `else`, but I haven't found a way to test it on my own [L180](https://github.com/pantheon-systems/documentation/pull/5739/files#diff-a43b4f6f8cd58b0d87a62563f4a1e2e5R180)

File: source/content/optimize-site-traffic.md [hunk 2]
```diff
@@ -174,9 +174,18 @@ The `stripos` function implements a case-insensitive match which can be helpful
 Remember to replace the example user agent (`UglyBot`):
 
 ```php:title=wp-config.php%20or%20settings.php
+// Either check a single bot.
 if (stripos($_SERVER['HTTP_USER_AGENT'], 'UglyBot') !== FALSE) {
   header('HTTP/1.0 403 Forbidden');
-  exit;
+}
+
+// Or check against a list of bots.
```
Comment (kyletaylored) [inline source/content/optimize-site-traffic.md:182]: Separated out the code blocks into separate blocks to stay away from multiple if/else and switch statements.

**Pull request merged**